apps/web/src/components/activity/ExecutionDetailsWebhookFeedback.tsx

```diff
@@ -20,30 +20,56 @@ const WebhookFeedbackTitle = styled(Text)`
   padding-top: 5px;
 `;
 
-const WebhookFeedback = ({ icon, text }) => {
+const WebhookTimeStamp = ({ timeStamp }) => {
+  const formattedDate = format(parseISO(timeStamp), 'dd/MM/yyyy');
+  const formattedTime = format(parseISO(timeStamp), 'HH:mm:ss');
+
+  return (
+    <span>
+      {formattedDate}, {formattedTime}
```

Comment:
Just curious in what was the reason to split the format date and time as they could go together in the same function call.

Reply:
Ah, right, added them at different point of time, and totally forgot date & time format exists 😂